packages/strapi-parts/src/Table/Table.js

```diff
@@ -46,6 +48,7 @@ const ScrollContainer = styled(Box)`
 export const Table = ({ colCount, rowCount, footer, ...props }) => {
   const tableRef = useRef(null);
   const [overflowing, setOverflowing] = useState();
+  console.log(footer);
```

Comment:
Can you remove the console.log please

Reply:
oh sure will do sorry!

Reply:
We should add an eslint rule for this, WDYT?